econml/solutions/causal_analysis/_causal_analysis.py

```diff
@@ -104,60 +105,67 @@ def _get_metadata_causal_insights_keys():
 
 
-def _first_stage_reg(X, y, *, automl=True):
+def _first_stage_reg(X, y, *, automl=True, random_state=None, verbose=0):
     if automl:
-        model = GridSearchCVList([make_pipeline(StandardScaler(), LassoCV()),
+        model = GridSearchCVList([make_pipeline(StandardScaler(), LassoCV(random_state=random_state)),
                                   RandomForestRegressor(
-                                      n_estimators=100, random_state=123, min_samples_leaf=10),
-                                  lgb.LGBMRegressor(num_leaves=32)],
+                                      n_estimators=100, random_state=random_state, min_samples_leaf=10),
+                                  lgb.LGBMRegressor(num_leaves=32, random_state=random_state)],
                                  param_grid_list=[{},
                                                   {'min_weight_fraction_leaf':
                                                       [.001, .01, .1]},
                                                   {'learning_rate': [0.1, 0.3], 'max_depth': [3, 5]}],
-                                 cv=2,
-                                 scoring='neg_mean_squared_error')
+                                 cv=3,
+                                 scoring='r2_score',
+                                 verbose=verbose)
         best_est = model.fit(X, y).best_estimator_
         if isinstance(best_est, Pipeline):
-            return make_pipeline(StandardScaler(), Lasso(alpha=best_est.steps[1][1].alpha_))
+            return make_pipeline(StandardScaler(), Lasso(alpha=best_est.steps[1][1].alpha_, random_state=random_state))
         else:
             return best_est
     else:
-        model = make_pipeline(StandardScaler(), LassoCV(cv=5)).fit(X, y)
-        return make_pipeline(StandardScaler(), Lasso(alpha=model.steps[1][1].alpha_))
+        model = make_pipeline(StandardScaler(), LassoCV(cv=5, random_state=random_state)).fit(X, y)
+        return make_pipeline(StandardScaler(), Lasso(alpha=model.steps[1][1].alpha_, random_state=random_state))
 
 
-def _first_stage_clf(X, y, *, make_regressor=False, automl=True):
+def _first_stage_clf(X, y, *, make_regressor=False, automl=True, random_state=None, verbose=0):
     if automl:
-        model = GridSearchCVList([make_pipeline(StandardScaler(), LogisticRegression(max_iter=1000)),
+        model = GridSearchCVList([make_pipeline(StandardScaler(), LogisticRegression(max_iter=1000,
+                                                                                     random_state=random_state,
+                                                                                     verbose=verbose)),
                                   RandomForestClassifier(
-                                      n_estimators=100, random_state=123),
-                                  GradientBoostingClassifier(random_state=123)],
+                                      n_estimators=100, random_state=random_state, verbose=verbose),
```

Comment:
I think we should remove the verbosity from the RF classifier and from the logistic regresison. Especially the RF classifier pringts too many things. 

For instance, we have not added these  verbosities in the regression case for the RF regressor. So also for uniformity

Reply:
Ok, done.